db/migrate/20250725050517_create_ticket_log_changes.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class CreateTicketLogChanges < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ticket_log_changes do |t|
+      t.references :ticket_log, null: false
+      t.string :field_name, null: false
+      t.string :field_value, null: false
+      t.timestamps
+    end
+
+    reversible do |dir|
+      dir.up do
+        TicketLog.find_each do |ticket_log|
+          if ticket_log.update_status?
```

Comment:
Reading this makes me think that `enum :update_status` in ticket log should probably be a `prefix: true`, so that this can read `if ticket_log.action_type_update_status?`

If that is too long for your taste, then instead of `prefix: true` as a boolean, you have the choice to actually specify a symbol as prefix: `prefix: :action` will give you `ticket_log.action_update_status?` for example